packages/salesforcedx-vscode-core/src/settings/sfdxCoreSettings.ts

```diff
@@ -77,10 +76,6 @@ export class SfdxCoreSettings {
     return this.getConfigValue(BETA_DEPLOY_RETRIEVE, false);
   }
 
-  public getApexLibrary(): boolean {
```

Comment:
We still need to remove the setting from the extension's `package.json` and its label from the `package.nls.json` & `package.nls.ja.json`